rel-eng/Makefile.python

```diff
@@ -1,32 +0,0 @@
-.DEFAULT_GOAL  :=  __unspecified
```

Comment:
I think we cannot remove this. It is still referenced in 15 other Makefiles.
I think it has a reason that it exist.

Reply:
And those seem still in use